setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='src',
+    packages=find_packages(),
+    version='0.1.0',
+    description='Predicting author from text',
+    author='Elaad Applebaum',
```

Comment:
Please add all of our names here.

Reply:
Added alphabetical by last name, though now am thinking maybe I should have done it in order of DART or KEND

Reply:
Thats alright. Thanks.